osu.Game/Rulesets/UI/Scrolling/ScrollingHitObjectContainer.cs

```diff
@@ -19,6 +19,16 @@ public class ScrollingHitObjectContainer : HitObjectContainer
         private readonly IBindable<double> timeRange = new BindableDouble();
         private readonly IBindable<ScrollingDirection> direction = new Bindable<ScrollingDirection>();
 
+        /// <summary>
+        /// Whether the scrolling direction is horizontal or vertical.
+        /// </summary>
+        private Direction scrollingAxis => direction.Value == ScrollingDirection.Left || direction.Value == ScrollingDirection.Right ? Direction.Horizontal : Direction.Vertical;
+
+        /// <summary>
+        /// Whether the scrolling direction is the positive-to-negative direction in the local coordinate.
```

Comment:
Shouldn't this be negative-to-positive? So if the scroll direction is down, an object is scrolling from 0 -> Length.

Reply:
My thought was the axis is inverted when if time of an object is increased then the position of the object is decreased.
I agree it could be understood in the opposite way and confusing.
Not sure how to express this.

Reply:
I was also similarly confused by this xmldoc, so I've reworded it a touch and added an example to hopefully explain better. See if you think that's more clear than what was here before.

(Disabling auto-merge pending confirmation.)